tests/rosetta.rs

```diff
@@ -0,0 +1,59 @@
+use std::{fs, path::Path};
+
+fn test_rosetta(dir_name: &str) {
+    let dir_path = Path::new("test-data").join(dir_name);
+    let expected = fs::read_to_string(dir_path.join("module.ron")).unwrap();
```

Comment:
I think this may need to be wrapped in `#[cfg(feature = "serialize")]` in order to get rid of unused var warnings